flytekit/clis/auth/credentials.py

```diff
@@ -1,12 +1,15 @@
 from __future__ import absolute_import
+
+import logging
```

Comment:
i'm guessing matt wants `as _logging`?

Reply:
@matthewphsmith katrina is calling you out on your pattern.

Reply:
indeed he does ;)